File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.ts
function maximumCount(nums: number[]): number {
     var maxCount = 0;
        var minCount = 0;

        for(var i = 0; i < nums.length; i ++){
            if (nums[i] < 0){
                minCount += 1;
            }
            if (nums[i] > 0){
                maxCount += 1;
            }
        }

    if (maxCount > minCount){
        return maxCount;
    }
    return minCount;
};

File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int maxCount = 0;
        int minCount = 0;

        for(int i = 0; i < nums.size(); i ++){
            if (nums[i] < 0){
                minCount += 1;
            }
            if (nums[i] > 0){
                maxCount += 1;
            }
        }

        return max(minCount, maxCount);
    }
};